egs/librimix/ConvTasNet/local/conf.yml

```diff
@@ -0,0 +1,33 @@
+# filterbank config
+filterbank:
+  n_filters: 512
+  kernel_size: 16
+  stride: 8
+# Network config
+masknet:
+  n_blocks: 8
+  n_repeats: 3
+  mask_act: relu
+  bn_chan: 128
+  skip_chan: 128
+  hid_chan: 512
+# Training config
+training:
+  epochs: 8
+  batch_size: 24
```

Comment:
This will work only on 4 GPUs, a more reasonnable default of 6 would be better.

Reply:
Sure